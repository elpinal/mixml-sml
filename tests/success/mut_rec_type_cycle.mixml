link X =
  link {module M = {type t : Type}} with
  {module N = {type t : Type}}
with
link
{ module M =
    link {type t : Type} seals {type t = ext X.N.t}
} with
{ module N =
    link {type t : Type} seals {type t = ext X.M.t}
}
