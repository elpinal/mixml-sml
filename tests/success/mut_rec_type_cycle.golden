{M : {t : [= !0]}, N : {t : [= !1]}}
